theme_loftspace/views/snippets/s_call_to_action.xml

```diff
@@ -12,14 +17,21 @@
         <xpath expr="//div[hasclass('col-lg-3')]" position="attributes">
             <attribute name="class" add="col-lg-12" remove="col-lg-3" separator=" "/>
         </xpath>
-        <!-- Paragraphs -->
+        <!-- Title & paragraphs -->
         <xpath expr="//h3" position="replace">
-            <h3>Would you like more information about a product?</h3>
+            <h3 style="font-size: 36px;">Would you like more information <br/>about a product?</h3>
         </xpath>
         <xpath expr="//p" position="replace"/>
         <!-- Button -->
         <xpath expr="//div[hasclass('col-lg-12')]/p" position="attributes">
             <attribute name="style" add="text-align: center;" remove="text-align: right;" separator=" "/>
         </xpath>
+        <xpath expr="//a[hasclass('btn')]" position="attributes">
+           <attribute name="class" remove="mb-2" separator=" "/>
+        </xpath>
```

Comment:
Normally, the mb-2 class is there for a reason and should not have an impact on your snippet? (it was added as a fix in case you duplicate the button, you want a vertical margin in mobile to have a space between your two buttons).